src/taoensso/sente.cljc

```diff
@@ -828,11 +829,12 @@
                    :requested-reconnect
                    :downgrading-ws-to-ajax
                    :unexpected}] reason)
-     (if (or (:open? state) (not= reason :unexpected))
-       (assoc state
-         :open? false
-         :last-close {:udt (enc/now-udt) :reason reason})
-       state)))
+     (let [state (dissoc state :next-reconnect)]
```

Comment:
Might be missing something, but is there a reason you'd want to drop the next reconnect info here?

To be clear, could you possibly briefly describe the intended behaviour. For example:

1. We set `:next-reconnect` any time we start an auto-reconnect timer
2. Under what circumstances would we want to remove this info?

My initial hunch would be to say on any reconnect, or on any :last-close reason change? Haven't had an opportunity to give it much though though.

Is it possible you wanted this dissoc 2 lines down, as part of the `then` branch?